src/tests/components/App.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../../components/App";
+
+describe("App", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
In this case this isnt actually needed since no mocks are being used, you could delete this